ui/components/multichain/account-picker/account-picker.js

```diff
@@ -37,24 +50,37 @@ export const AccountPicker = ({ address, name, onClick, disabled }) => {
       }}
       disabled={disabled}
     >
-      <AvatarAccount
-        variant={
-          useBlockie
-            ? AvatarAccountVariant.Blockies
-            : AvatarAccountVariant.Jazzicon
-        }
-        address={address}
-        size={Size.XS}
-        borderColor={BackgroundColor.backgroundDefault} // we currently don't have white color for border hence using backgroundDefault as the border
-      />
-      <Text as="span" fontWeight={FontWeight.Bold} ellipsis>
-        {name}
-      </Text>
-      <Icon
-        name={IconName.ArrowDown}
-        color={IconColor.iconDefault}
-        size={Size.SM}
-      />
+      <Box display={Display.Flex} flexDirection={FlexDirection.Column}>
+        <Box>
```

Comment:
The default display of the `Box` is `block` so here we need to add flex-box. You could use something like
```suggestion
        <Box display={Display.Flex} alignItems={AlignItems.center} gap={1}>
```